cmd/dockerd/daemon.go

```diff
@@ -342,6 +343,26 @@ func shutdownDaemon(d *daemon.Daemon) {
 	}
 }
 
+// Previously we just documented that the com.docker.*, io.docker.*, org.dockerproject.*
+// namespaces were reserved and should not be used, but we did not enforce this.  Now
+// we will filter them out, but we should error instead at the next stable release.
+// The logic for erroring can probably be added to config.Validate.
+func filterReservedNamespaceLabels(labels []string) []string {
+	var filtered []string
+	for _, label := range labels {
+		lowered := strings.ToLower(label)
+		if strings.HasPrefix(lowered, "com.docker.") || strings.HasPrefix(lowered, "io.docker.") ||
+			strings.HasPrefix(lowered, "org.dockerproject.") {
+			continue
+		}
+		filtered = append(filtered, label)
+	}
+	if len(filtered) < len(labels) {
+		logrus.Warn("The com.docker.*, io.docker.*, and org.dockerprojects.* namespaces are reserved.  Labels using these namespaces will be ignored.")
```

Comment:
nit: 

- there's two spaces before `Labels`
- perhaps `s/will be ignored/are ignored/`

Wondering if during the deprecation period we should still apply the labels, and only print the warning. Although "unlikely", there _may_ be someone somewhere actually using labels with this prefix 😞

Reply:
Perhaps it's okay, given that this was already documented; would like some others to kick in on this though 😅

Reply:
I do agree with @thaJeztah and I think there *is* someone using labels with those prefixes.